test/src/main/java/org/apache/accumulo/test/functional/FateConcurrencyIT.java

```diff
@@ -135,7 +136,7 @@ public void changeTableStateTest() throws Exception {
     OnlineOpTiming timing1 = task.get();
 
     log.trace("Online 1 in {} ms",
-        TimeUnit.MILLISECONDS.convert(timing1.runningTime(), TimeUnit.NANOSECONDS));
+        MILLISECONDS.convert(timing1.runningTime(), NANOSECONDS));
```

Comment:
`.convert()` is a very confusing, because it's not obvious which direction it is converting. The following would be better for these:

```suggestion
        NANOSECONDS.toMillis(timing1.runningTime()));
```

Reply:
This note applied to the other occurrences in the same file (and possibly elsewhere).